postmortems/2024-04-01-tx-receipts-validation-skip-incident.md

```diff
@@ -0,0 +1,61 @@
+# 2024-04-03 Chain Halt due to tx receipts validation skip
```

Comment:
```suggestion
# 2024-04-01 Chain Halt due to Tx Receipts Validation Skip Post-Mortem
```

Reply:
Thanks, I applied in [47009b1](https://github.com/kroma-network/kroma/pull/289/commits/47009b1023eb7a05e741326b5c955428aa1ebe88)